benchmarks/templates/benchmarks/tutorial_submitted.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'benchmarks/base.html' %}
+{% load static %}
+{% load compress %}
+
+{% block main %}
+<nav class="navbar updated is-fixed-top is-transparent">
+    <div class= "container my_container nav-container">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="https://www.brain-score.org">
+                    <img class="navbar_element image" id="navbar-logo-colored" src="/static/benchmarks/img/colored_logo.png"/>
+            </a>
+            <a role="button" class="navbar-burger" data-target="navMenu" aria-label="menu" aria-expanded="false">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navMenu" class="navbar-menu">
+                <div class="navbar-end">
+                    <div class="navbar-item">
+                        <a class="social_icon navbar-item " href="https://www.github.com/brain-score">
+                            <img class="navbar_element image" id="github_icon" src="/static/benchmarks/img/github_icon.png" />
+                        </a>
+                        <a class="social_icon navbar-item" href="https://www.twitter.com/brain_score">
+                            <img class="navbar_element image" id="twitter_icon" src="/static/benchmarks/img/twitter_icon.png"/>
+                        </a>
+                    </div>
+                </div>
+        </div>
+    </div>
+</nav>
+<div class="container login" style="background-image: url({% static '/benchmarks/img/login_brain.png' %});">
+    <div class="column is-half has-text-centered-mobile">
+        <div class="container login-left">
+            <section id="success" class="container center">
+                <article class="message">
+                    <div class="message-header">
+                        <p>Error: Submitted Tutorial Model </p>
+                    </div>
+                    <div class="message-body">
+                        It looks like you have tried to submit the model from our tutorial.
+                        Please do not upload tutorial plugins, as they will get merged into our code base and cause
+                        duplicate entries.
```

Comment:
shouldn't we say that the model already exists? let's just use the regular resnet in the tutorial so that the submission will be rejected, and then instruct the user here on what they should do.